app/src/main/java/org/mozilla/focus/home/pocket/Pocket.kt

```diff
@@ -110,7 +110,7 @@ object Pocket {
 data class PocketVideo(
     val id: Int,
     val title: String,
-    val url: String,
+    val url: String?,
```

Comment:
I don't like representing `PocketVideoPlaceholder` as a `PocketVideo` with a null URL, especially if it's undocumented: I'd prefer to use a sealed class, a "PocketVideo" kind enum, or similar.